docs/proposal/observability.md

```diff
@@ -0,0 +1,216 @@
+any review.
+-->
+
+### Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap.
+
+A good summary is probably at least a paragraph in length.
+-->
+
+The importance of observability of service mesh as the basis for manageable, reliable and sustainable grid systems cannot be ignored.In istio, accesslog, metric, and tracing are provided at layers l4 and l7 to meet the user's need for observability.
+
+In this proposal, I will analyse the observability metrics of istio. And propose that Kmesh implement observability features to support these metrics. So that users can seamlessly use Kmesh.
+
+### Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users.
+-->
+
+#### Accesslog
+
+In [istio ztunnel](https://github.com/istio/ztunnel?tab=readme-ov-file#logging), the Layer 4 access log contains the following metrics:
+
+src.addr
+src.workload
+src.namespace
+src.identity
+
+dst.addr
+dst.hbone_addr
+dst.service
+dst.workload
+dst.namespace
+dst.identity
+
+direction
+
+bytes_sent
+bytes_recv
+duration
+
+An example of the accesslog obtained is shown below:
+
+```console
+2024-04-11T15:38:42.182974Z  INFO access: connection complete
+    src.addr=10.244.0.24:46238 src.workload="shell-6d8bcd654d-t88gp" src.namespace="default" src.identity="spiffe://cluster.local/ns/default/sa/default"
+    dst.addr=10.244.0.42:15008 dst.hbone_addr=10.96.108.116:80 dst.service="echo.default.svc.cluster.local"
+    direction="outbound" bytes_sent=67 bytes_recv=490 duration="13ms"
+```
+
+The accesslog needs to contain the identities(address/workload/namespace/identity) of the destination and source. In addition, the required metrics are the size of the message sent(bytes_sent), the size of the message received(bytes_recv) and the duration of the link.
+
+In order for users to be able to use Kmesh smoothly, Kmesh needs to support these accesslog.
+
+#### Metrics
+
+To monitor service behavior, Istio generates metrics for all service traffic in, out, and within an Istio service mesh. These metrics provide information on behaviors.
+
+Refer to [istio ztunnel metric](https://github.com/istio/ztunnel/blob/6532c553946856b4acc326f3b9ca6cc6abc718d0/src/proxy/metrics.rs#L369) , at Layer L4, the required metric is:
+
+```console
+connection_opens: The total number of TCP connections opened
+connection_close: The total number of TCP connections closed
+received_bytes: The size of total bytes received during request in case of a TCP connection
+sent_bytes: The size of total bytes sent during response in case of a TCP connection
+on_demand_dns: The total number of requests that used on-demand DNS (unstable)
+on_demand_dns_cache_misses: The total number of cache misses for requests on-demand DNS (unstable)
```

Comment:
some metrics like sent_bytes should have some labels, please double check what does it look like, and socument it here